prototype/wof.js

```diff
@@ -148,10 +148,10 @@ function getPopulation( wof ) {
 }
 
 // abbreviations and ISO codes
-// logic copied from: pelias/whosonfirst src/components/extractFields.js
+// logic copied from: pelias/whosonfirst src/components/extractFields.js (since modified)
 function getAbbreviation( wof ) {
   if( 'country' === wof['wof:placetype'] || 'dependency' === wof['wof:placetype'] ) {
-    return wof['wof:country_alpha3'];
+    return wof['wof:country_alpha3'] || wof['wof:abbreviation'];
```

Comment:
looking at a few dependencies in spelunker, `wof:country_alpha3` is the 3-letter code but `wof:abbreviation` is always the 2-letter version. Perhaps there is a different field we can use or ask the data to team to add `wof:country_alpha3` to all dependency records for consistency? Also, not all dependencies have either `wof:country_alpha3` or `wof:abbreviation` so we would still have the issue of no country code in those records.